### PR TITLE
framework/ble_manager: fix BLE build warnings

### PR DESCRIPTION
**Pull request by an4967**
- fix build warnings in BLE example
- fix BLE manager bug
- add default value in BT Kconfig

### CODE REVIEW

File: framework/src/ble_manager/ble_manager_state.c [hunk 1]
```diff
@@ -913,6 +913,7 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		if (ctx == NULL) {
 			BLE_LOG_ERROR("[BLEMGR] fail to find BLE context table");
 			ret = TRBLE_NOT_FOUND;
+			free(msg->param);
```
Comment (jeongchanKim) [inline framework/src/ble_manager/ble_manager_state.c:916]: In line 932, tries to free the `msg->param` again.
Please check this.

Comment (jeongchanKim) [inline framework/src/ble_manager/ble_manager_state.c:916]: From here, we don't use `msg->param` anymore.
How about free it befor this if statement?
Then we don't need to call free from line 932

Comment (an4967) [reply framework/src/ble_manager/ble_manager_state.c:916]: @jeongchanKim 
The value of 'msg->param'  is allocated in 'ble_device_connected *data'.
It keeps using until the end of 'BLE_EVT_CLIENT_CONNECT' case

Comment (sangam-swami) [inline framework/src/ble_manager/ble_manager_state.c:916]: we can separate commits for specific fixes?

Comment (an4967) [reply framework/src/ble_manager/ble_manager_state.c:916]: Okay, I separated it into 3 commits

File: apps/examples/ble_rmc/ble_rmc_main.c [hunk 2]
```diff
@@ -96,6 +96,19 @@ static void ble_device_scanned_cb_with_filter(ble_scanned_device *scanned_device
 	}
 }
 
+static void ble_device_scanned_cb_with_filter(ble_scanned_device *scanned_device)
+{
+	RMC_LOG(RMC_CLIENT_TAG, "Find RMC with scan filter!!!\n");
+	printf("scanned mac : %02x:%02x:%02x:%02x:%02x:%02x\n", 
+		scanned_device->addr.mac[0],
```
Comment (pillip8282) [inline apps/examples/ble_rmc/ble_rmc_main.c:103]: why did you separate log? RMC_LOG and printf

File: apps/examples/ble_rmc/ble_rmc_main.c [hunk 3]
```diff
@@ -447,15 +460,16 @@ int ble_rmc_main(int argc, char *argv[])
 
 		if (argc == 4 && strncmp(argv[2], "del", 4) == 0) {
 			int cnt = 0;
-			uint8_t mac[BLE_BD_ADDR_MAX_LEN] = { 0, };
+			ble_addr addr[1] = { 0, };
```
Comment (pillip8282) [inline apps/examples/ble_rmc/ble_rmc_main.c:463]: it's good idea to make char[] to struct. it's easy to read.

Comment (an4967) [reply apps/examples/ble_rmc/ble_rmc_main.c:463]: Thanks a lot

File: apps/examples/ble_rmc/ble_rmc_main.c (hunk 2 above)
Comment (an4967) [reply apps/examples/ble_rmc/ble_rmc_main.c:103]: Okay, I will modify this in next PR.

**Review by pillip8282 — APPROVED** [review]
LGTM

**Pull request merged**